src/backend/app/app.py

```diff
@@ -57,13 +112,13 @@ def login():
         email = request.form['email']
         password = request.form['password']
 
-        # Retrieve the user from the database
```

Comment:
why did u delete my comments ?

Reply:
I might have done that by mistake. Can you add again?